.github/workflows/publish.yml

```diff
@@ -18,6 +18,10 @@ on:
         description: "Visibility of the published specs. Can be 'public', 'partner' or 'internal'."
         type: string
         default: "public"
+      validate_only:
+        description: "If true, will not upload the spec(s) to a GCS bucket for deploy to Developer Portal, but only validate that the upload is allowed."
```

Comment:
Jeg synes heller ikke at vår interne infrastruktur trengs å beskrives her. Hva hjelper det utviklern? "If true, will not publish the spec(s) to the Developer portal, but only validate that they can be published".